modules/claims_api/config/routes.rb

```diff
@@ -44,6 +44,11 @@
       post '/:veteranId/claims/:id/5103', to: 'evidence_waiver#submit'
       ## 2122 Forms
       get '/:veteranId/power-of-attorney', to: 'power_of_attorney#show'
+      post '/:veteranId/2122/validate', to: 'power_of_attorney#validate_2122'
+      post '/:veteranId/2122', to: 'power_of_attorney#submit_2122'
+      post '/:veteranId/2122a/validate', to: 'power_of_attorney#validate_2122a'
+      post '/:veteranId/2122a', to: 'power_of_attorney#submit_2122a'
+      get '/:veteranId/power-of-attorney/:poaCode', to: 'power_of_attorney#status'
```

Comment:
I'd go with an `:id` here instead of `:poaCode` as the PoA Code is more a code associated to a particular PoA - refer to v1 documentation for that reference - where this is the ID to check the status of a specific PoA Request.